lib/ocnless_cluster.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require "clusterable/holding"
+require "clusterable/ht_item"
+require "clusterable/commitment"
+require "clusterable/ocn_resolution"
+require "calculate_format"
+require "clustering/cluster_ht_item"
+require "cluster_error"
+
+# A set of identifiers (e.g. OCLC numbers),
+# - ocns
+# - holdings
+# - htitems
+# - commitments
+class OCNLessCluster
+  attr_reader :id
+
+  def initialize(bib_key:)
+    @bib_key = bib_key
+  end
+
+  def ocns
```

Comment:
Should just always return an empty array?

Reply:
`Cluster.new.ocns.class` returns Set. Behavior for this class chosen to parallel that of `Cluster` (in this case at least LOL)

Reply:
Done.